### ISSUE TITLE
[BUG] Fixed top Sidebar eats space from story river

### ISSUE THREAD

**Issue by danielo515** [Jermolene/TiddlyWiki5#1911]
Hello,

In the latest pre-release of tiddlywiki the sidebar is obscuring the first tiddler of the story river when you go to stretch mode.

Steps to reproduce:
* Go to http://tiddlywiki.com/prerelease/
* Make the window less than 900px width
* You can not see about 50% of the first tiddler

Regards

**Comment by danielo515** [comment]
I can confirm that this problem is **very** noticeable in mobile. In fact, it makes TW absolutely unusable on mobile devices of less than 8". @Jermolene , any chance to prioritize this?

**Comment by Jermolene** [comment]
Hi @danielo515 this is an incomplete experiment that you can see here:

cac6d3b4d92908915ade3fe22df263da2b903557

The goal was to explore making the sidebar available without having to scroll to the top of the story river. As the commit notes, a big problem at the moment is that the sidebar is visible by default. If you try switching the sidebar off then you might see the purpose of the experimental arrangement.

So, I think that what remains to be done is:

1. Make the sidebar hidden by default on narrow screens
2. Make the sidebar full screen when it is visible

**Comment by danielo515** [comment]
Hello @Jermolene 

Hope you don't find this offensive, but I found this as a fast but non optimal solution to the mobile problem. The sidebar is not optimized in any way to be usable on mobile screens, so all what we get is the same Desktop interface but at a fixed position:

* Buttons remain small
* Links remain hard to click on
* Search bar still hard to reach

This is even uncomfortable on desktop windows where you have it small enough to trigger the sidebar going to top, because you end with half of the screen hidden in a way yo don't expect. Honestly, it looks more like a bug than a feature, at least I took it as a bug at first instance.
